src/Model/Consignment/DropOffPoint.php

```diff
@@ -255,7 +255,7 @@ public function setNumber(string $number): self
      */
     public function setNumberSuffix(?string $numberSuffix = null): self
     {
-        $this->number_suffix = $numberSuffix;
+        $this->number_suffix = $numberSuffix ?? '';
```

Comment:
Wat is de originele waarde? Als je `getNumberSuffix` doet voordat of zonder je `setNumberSuffix` doet krijg je alsnog een fatal error